app/src/main/java/org/mifos/mobile/models/AboutUsItem.kt

```diff
@@ -0,0 +1,9 @@
+package org.mifos.mobile.models
+
+import org.mifos.mobile.ui.enums.AboutUsListItemId
+
+data class AboutUsItem(
+    val name: Int,
```

Comment:
Rename this to the `title` and add another item named `subtitle` (You would need this for the copyright item )

Reply:
done